collection-scripts/version

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+function version() {
+  # get version from odh/rhods operator image
+  version=''
+  operator_name=$(oc get pod -n redhat-ods-operator --ignore-not-found |grep rhods-operator |awk '{print $1}')
+
+  [[ z != z"${operator_name}" ]] && version=$(oc exec ${operator_name} -n redhat-ods-operator -c rhods-operator -- env|grep OPERATOR_CONDITION_NAME|awk -F'=' '{print $2}'|     sed -n -r -e 's/.*([[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+(:?\.[[:digit:]])?(:?-[^@]+)?).*/\1/p')
+
+  # if version not found, get version from odh/rhods csv
+  csv_name=$(oc get csv|grep rhods|awk '{print $1}')
+  [[ z == z"${version}" && z != z"${csv_name}" ]] && version=$(oc get csv $(oc get csv|grep rhods|awk '{print $1}') -o json|jq .spec.version |xargs)
```

Comment:
out of my curiosity, when the previous method could fail? Wouldn't be enough to take the version from the csv?

Reply:
It is just in case. I also think the csv must be there so I believe the second function can get the right version.